.idea/gradle.xml

```diff
@@ -4,7 +4,7 @@
   <component name="GradleSettings">
```

Comment:
lets add all the `.idea` files to `.gitignore` as we dont need them

Reply:
sure.